src/demo/demo.test.tsx

```diff
@@ -1,6 +1,12 @@
-import { h } from 'preact'
+import { h, FunctionComponent } from 'preact'
 import { mount } from 'enzyme'
 
+declare global {
+  interface Window {
+    domNode: HTMLElement
+  }
+}
+
```

Comment:
What happens if you remove this?

Reply:
The TypeScript compiler will complain:

```
 FAIL  src/demo/demo.test.tsx
  ● Test suite failed to run

    src/demo/demo.test.tsx:40:12 - error TS2339: Property 'domNode' does not exist on type 'Window & typeof globalThis'.

    40     window.domNode = rootNode
                  ~~~~~~~
```

Reply:
Makes sense, thanks for the clarification